Documentation/contributing/development/hive.rst

```diff
@@ -903,7 +903,7 @@ test against and allows central control over what data (and at what rate)
 is pulled from the api-server and how it’s stored (in-memory or persisted).
 
 The resources are usually made available centrally for the application,
-e.g. in cilium-agent they’re provided from `pkg/k8s/shared_resources.go <https://github.com/cilium/cilium/blob/main/pkg/k8s/shared_resources.go>`_.
+e.g. in cilium-agent they’re provided from `pkg/k8s/resource.go <https://github.com/cilium/cilium/blob/main/pkg/k8s/resource/resource.go>`_.
```

Comment:
Not sure about this one, @pippolo84 could you please confirm this is the right link to use?

Reply:
The correct link is this one: https://github.com/cilium/cilium/blob/main/daemon/k8s/resources.go